example/rocketmq_v1alpha1_console_cr.yaml

```diff
@@ -1,11 +1,47 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: console-config
+data:
+  application.yml: |
+    server:
+      port: 8080
+      servlet:
+        encoding:
+          charset: UTF-8
+          enabled: true
+          force: true
+    spring:
+      application:
+        name: rocketmq-dashboard
+    logging:
+      config: classpath:logback.xml
+    rocketmq:
+      config:
+        namesrvAddrs:
+          - XXX:XXX
```

Comment:
IMO nameserver address should be injected by rocketmq-operator, not manually configured.

Reply:
Ok

Reply:
@shendongsd Should we also define `role-permission.yml` in config map?

Reply:
The role-permission.yml can be added to configmap

Reply:
@shendongsd Pls also add `role-permission.yml` to config map and reference it in `Console` CR.

Reply:
Ok，I could add  it later